classes/aws_elastic_transcoder.php

```diff
@@ -197,4 +197,32 @@ public function get_presets(array $presetids=array()) : array {
         }
         return $presets;
     }
+
+     /**
+     * Return an array of all preset ids that the plugin can use.
+     *
+     * @return array $presetids The preset ids.
+     */
+    public function get_all_preset_ids() : array {
+        $presetids = [];
+
+        // Collate enabled presets.
+        $presetids = array_merge(self::LOW_PRESETS, $presetids);
```

Comment:
I think could read nicer as just

```
array_merge(
    self::LOW_PRESETS,
    self::MEDIUM_PRESETS,
    self::HIGH_PRESETS,
    ...
);

Reply:
and then its probably simple enough to merge these two functions

Reply:
Yeah it can be better. I did it this way as it reads the same as the above merge, but it can be much simpler.